integration/config/setup.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+kubectl label namespace default foo=bar
```

Comment:
Leftover?

Reply:
It's just some dummy commands to test that the setup script param works.

Reply:
For me it would make sense to move this to the `architect` orb then and use it from there. Or do you want to merge it here?

Reply:
Yes sure I'll move it.

It can't be merged here. It's just the inline orb is a faster feedback loop if we wanted to make changes.